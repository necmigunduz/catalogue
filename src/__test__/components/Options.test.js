import React from 'react';
import renderer from 'react-test-renderer';
import Option from '../../components/Options';

describe('Option', () => {
    it('renders correctly', () => {
      const options = ['1', '2', '3'].map((option) => <option key={option}>{option}</option>);
      const tree = renderer
        .create(<Option
          id="dishType"
          options={options}
          value="4"
        />)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
