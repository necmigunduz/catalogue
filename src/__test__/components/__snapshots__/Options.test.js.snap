// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Option renders correctly 1`] = `
<div>
  <select
    id="dishType"
    value="4"
  >
    <option>
      1
    </option>
    <option>
      2
    </option>
    <option>
      3
    </option>
  </select>
  <div />
</div>
`;
